src/avram/postgres_url.cr

```diff
@@ -1,39 +1,72 @@
 class Avram::PostgresURL
-  getter database, username, password, hostname, port
+  getter database, username, password, hostname, port, query
+  property url : String = ""
 
   def initialize(
     @database : String,
-    @hostname : String,
+    @hostname : String = "",
     @username : String = "",
     @password : String = "",
-    @port : String = ""
+    @port : String = "",
+    @query : String = ""
   )
   end
 
-  def self.build(**args)
+  def self.build(**args) : PostgresURL
     new(**args).build
   end
 
+  def self.parse(nothing : Nil)
```

Comment:
I'm not sure if this is a crystal convention, but generally I'll put in an underscore if I never reference the param:
```suggestion
  def self.parse(_ : Nil)
```

Reply:
```
In src/avram/postgres_url.cr:19:19

 19 | def self.parse(_ : Nil)
                      ^
Error: unexpected token: UNDERSCORE
```

Turns out you can't do that 😂

Reply:
Boo. What about `_nothing`?